### PR TITLE
CV ontology terms

### PR DESCRIPTION
**Pull request by ypriverol**
Some refinements suggested in the following Issue https://github.com/bigbio/proteomics-metadata-standard/issues/33#issuecomment-620504049

### CODE REVIEW

File: pride_cv.obo [hunk 1]
```diff
@@ -3720,69 +3722,65 @@ id: PRIDE:0000557
 name: Size-exclusion chromatography (SEC)
 def: "A chromatography method where the separation is caused by differences in molecular size." []
 xref: http://purl.obolibrary.org/obo/CHMO_0001013
-is_a: PRIDE:0000550 ! Fractionation method
-is_a: PRIDE:0000551 ! Separation method
+is_a: PRIDE:0000556 ! Ion-exchange chromatography
```
Comment (mwalzer) [inline pride_cv.obo:3725]: agreed, I think putting it underneath family relation is the best solution

File: pride_cv.obo [hunk 2]
```diff
@@ -3720,69 +3722,65 @@ id: PRIDE:0000557
...
 xref: http://purl.obolibrary.org/obo/CHMO_0002302
-is_a: PRIDE:0000563 ! Reversed-phase chromatography (RP)
+is_a: PRIDE:0000556 ! Ion-exchange chromatography
 
 [Term]
 id: PRIDE:0000565
 name: High-performance liquid chromatography
 def: "Column chromatography where the mobile phase is a liquid, the stationary phase consists of very small particles and the inlet pressure is relatively high." []
 xref: http://purl.obolibrary.org/obo/CHMO_0001009
+is_a: PRIDE:0000550 ! Fractionation method
```
Comment (mwalzer) [inline pride_cv.obo:3781]: Yep, they were missing

File: pride_cv.obo [hunk 3]
```diff
@@ -3720,69 +3722,65 @@ id: PRIDE:0000557
...
-is_a: PRIDE:0000551 ! Separation method
+is_a: PRIDE:0000556 ! Ion-exchange chromatography
 
 [Term]
 id: PRIDE:0000564
 name: High-pH reversed-phase chromatography (hpHRP)
 def: "Liquid chromatography where the mobile phase is of high pH and significantly more polar than the stationary phase." []
 xref: http://purl.obolibrary.org/obo/CHMO_0002302
-is_a: PRIDE:0000563 ! Reversed-phase chromatography (RP)
+is_a: PRIDE:0000556 ! Ion-exchange chromatography
```
Comment (mwalzer) [inline pride_cv.obo]: Was this on purpose?  (RP and hpHRP are now on same hierarchy level) If so, fine with me.

**Review by mwalzer — COMMENTED** [review]
Check the second to last comment. If that was deliberate, then I think we are fine to merge

**Comment by ypriverol** [comment]
@mwalzer I solve all the issues!!!! Please have a look.

**Pull request merged**